scripts/empty_upgrade_handler_gen.sh

```diff
@@ -81,15 +82,40 @@ func CreateUpgradeHandler(
 	}
 }" >> $UPGRADES_FILE
 
-  # change app/app.go file
- app_file=./app/app.go
- UPGRADES_LINE=$(grep -F upgrades.Upgrade{ $app_file)
- UPGRADES_LINE="${UPGRADES_LINE%?}, ${version_create}.Upgrade}"
- sed -i "s|.*upgrades.Upgrade{.*|$UPGRADES_LINE|" $app_file 
+# change app/app.go file
+app_file=./app/app.go
+UPGRADES_LINE=$(grep -F upgrades.Upgrade{ $app_file)
+UPGRADES_LINE="${UPGRADES_LINE%?}, ${version_create}.Upgrade}"
+sed -i "s|.*upgrades.Upgrade{.*|$UPGRADES_LINE|" $app_file 
+
+PREV_IMPORT="v$latest_version $module/app/upgrades/v$latest_version$bracks"
+NEW_IMPORT="$version_create $module/app/upgrades/$version_create$bracks"
+sed -i "s|.*$PREV_IMPORT.*|\t$PREV_IMPORT\n\t$NEW_IMPORT|" $app_file
+
+# change e2e version in makefile
+sed -i "s/E2E_UPGRADE_VERSION := ${bracks}v$latest_version$bracks/E2E_UPGRADE_VERSION := ${bracks}$version_create$bracks/" ./Makefile
+
+# bumps up prev e2e version
+e2e_file=./tests/e2e/containers/config.go
+PREV_OSMOSIS_DEV_TAG=$(curl -L -s 'https://registry.hub.docker.com/v2/repositories/osmolabs/osmosis-dev/tags?page=1&page_size=100'            | jq -r '.results[] | .name | select(.|test("^(?:v|)[0-9]+\\.[0-9]+(?:$|\\.[0-9]+$)"))' | grep -m1 "")
+PREV_OSMOSIS_E2E_TAG=$(curl -L -s 'https://registry.hub.docker.com/v2/repositories/osmolabs/osmosis-e2e-init-chain/tags?page=1&page_size=100' | jq -r '.results[] | .name | select(.|test("^(?:v|)[0-9]+\\.[0-9]+(?:$|\\.[0-9]+$)"))' | grep -m1 "")
```

Comment:
Hey @MasterPi-2124! Thanks for the PR. I might be missing something, but shouldn't it be like this? Not a pro in grep flags so correct me if I am wrong :)
```suggestion
PREV_OSMOSIS_E2E_TAG=$(curl -L -s 'https://registry.hub.docker.com/v2/repositories/osmolabs/osmosis-e2e-init-chain/tags?page=1&page_size=100' | jq -r '.results[] | .name | select(.|test("^(?:v|)[0-9]+\\.[0-9]+(?:$|\\.[0-9]+$)"))' | grep -m 1 "")
```

Reply:
actually both are correct, but for clarity for other people, I will change that into the full flag

Reply:
got it! thanks for the clarification